### PR TITLE
[NCP-3679] improve the message when creating/updating is submitted

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

File: zh-Hans.json [hunk 1]
```diff
@@ -710,7 +710,7 @@
 	"edgeHostname.confirm.cancel.create.edgeHostname": "是否要退出创建此调度域名？",
 	"edgeHostname.confirm.cancel.edit.edgeHostname": "您是否要放弃更新此调度域名？",
 	"edgeHostname.create.success.modal.content": "调度域名已成功创建。",
-	"edgeHostname.create.success.modal.content.2": "边缘主机名已成功创建。边缘主机名需要几秒钟才能生效。您可以打开详细信息页面并在 {history} 下拉列表中查看状态。",
+	"edgeHostname.create.success.modal.content.2": "边缘主机名已成功创建。边缘主机名生效需要几分钟时间。您可以打开详细信息页面并在 {history} 下拉列表中检查状态。请确保状态为成功且边缘主机名已生效，然后才能在您的 DNS 记录中使用边缘主机名。",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:713]: Please change it to:

调度域名已成功创建。调度域名的部署需要几分钟时间。您可以打开详请页面并在 {操作历史} 下拉列表中检查部署状态。请确保域名已部署成功且配置生效，才在DNS记录中使用该调度域名。

File: zh-Hans.json [hunk 2]
```diff
@@ -782,7 +782,7 @@
 	"edgeHostname.tooltip.client.zone.servergroup": "设置服务该访客分区的服务集群的最高等级。这将启用所有不高于该等级的节点。",
 	"edgeHostname.tooltip.client.zone.weight": "设置该规则在其所在访客分区里生效的比例。",
 	"edgeHostname.tooltip.server.regions": "选择可以为调度域名提供服务的区域。",
-	"edgeHostname.update.accepted.modal.content": "边缘主机名更新已接受。更改需要几秒钟才能生效。您可以在 {history} 下拉列表中检查状态。",
+	"edgeHostname.update.accepted.modal.content": "边缘主机名更新已接受。更改生效需要几分钟时间。您可以在 {history} 下拉列表中检查状态。在 DNS 记录中使用边缘主机名之前，请检查状态是否为“成功”且更改是否已生效。",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:785]: Please change it to:

调度域名更新请求已接受。调度域名的部署需要几分钟时间。您可以打开详请页面并在 {操作历史} 下拉列表中检查部署状态。请确保域名已部署成功且配置生效，才在DNS记录中使用该调度域名。

**Review by Jianghai-Quantil — CHANGES_REQUESTED** [review]
Requested some changes

File: en_US.json [hunk 3]
```diff
@@ -782,7 +782,7 @@
 	"edgeHostname.tooltip.client.zone.servergroup": "Select the highest server group to use.  Your traffic will be served by all PoPs not higher than it.",
 	"edgeHostname.tooltip.client.zone.weight": "Adjust how this rule behaves relative to other rules for the same client zone.",
 	"edgeHostname.tooltip.server.regions": "Select the region(s) that can serve the edge hostname.",
-	"edgeHostname.update.accepted.modal.content": "The edge hostname update has been accepted. It will take a few seconds before the change takes effect. You can check the status in the {history} dropdown list.",
+	"edgeHostname.update.accepted.modal.content": "The edge hostname update has been accepted. It will take a few minutes before the change takes effect. You can check the status in the {history} dropdown list. Please check that the status ‘Succeeded’ and that the change has taken effect, before using the edge hostname in your DNS records.",
```
Comment (burtonquantil) [inline en_US.json:785]: Sorry for the confusion. Add the word "is":

 **Please check that the status is ‘Succeeded’**

File: en_US.json [hunk 4]
```diff
@@ -710,7 +710,7 @@
 	"edgeHostname.confirm.cancel.create.edgeHostname": "Do you want to quit creating this edge hostname?",
 	"edgeHostname.confirm.cancel.edit.edgeHostname": "Do you want to discard the update of this edge hostname?",
 	"edgeHostname.create.success.modal.content": "The edge hostname has been successfully created.",
-	"edgeHostname.create.success.modal.content.2": "The edge hostname has been successfully created. It will take a few seconds before the edge hostname takes effect. You can open the details page and check the status in the {history} dropdown list.",
+	"edgeHostname.create.success.modal.content.2": "The edge hostname has been successfully created. It will take a few minutes before the edge hostname takes effect. You can open the details page and check the status in the {history} dropdown list. Please make sure that the status is succeeded and that the edge hostname has taken effect, before using the edge hostname in your DNS records.",
```
Comment (burtonquantil) [inline en_US.json:713]: Change one word:

**Please make sure that the status is ‘Succeeded’**

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Minor text change requested

**Comment by jungho73** [comment]
@burtonquantil @Jianghai-Quantil 
I updated some text again according to comments from the issue .
Please check again your changes requested.

**Review by burtonquantil — APPROVED** [review]
The English text is OK.

**Comment by Jianghai-Quantil** [comment]
@jungho73 The Chinese texts are not changed as requested. Can you double check?

**Comment by jungho73** [comment]
> @jungho73 The Chinese texts are not changed as requested. Can you double check?

@Jianghai-Quantil  Since the English sentence changed, I thought the Chinese you requested should be changed first. First, I made the corrections you requested. Please review again.

File: zh-Hans.json [hunk 5]
```diff
@@ -710,7 +710,7 @@
 	"edgeHostname.confirm.cancel.create.edgeHostname": "是否要退出创建此调度域名？",
 	"edgeHostname.confirm.cancel.edit.edgeHostname": "您是否要放弃更新此调度域名？",
 	"edgeHostname.create.success.modal.content": "调度域名已成功创建。",
-	"edgeHostname.create.success.modal.content.2": "边缘主机名已成功创建。边缘主机名需要几秒钟才能生效。您可以打开详细信息页面并在 {history} 下拉列表中查看状态。",
+	"edgeHostname.create.success.modal.content.2": "调度域名已成功创建。调度域名的部署需要几分钟时间。您可以打开详请页面并在 {history} 下拉列表中检查部署状态。请确保域名已部署成功且配置生效，才在DNS记录中使用该调度域名。",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:713]: Please replace {history} with {操作历史}.

When the language selected by user is Chinese, there won't be the label for the dropdown won't History.

File: zh-Hans.json [hunk 6]
```diff
@@ -782,7 +782,7 @@
 	"edgeHostname.tooltip.client.zone.servergroup": "设置服务该访客分区的服务集群的最高等级。这将启用所有不高于该等级的节点。",
 	"edgeHostname.tooltip.client.zone.weight": "设置该规则在其所在访客分区里生效的比例。",
 	"edgeHostname.tooltip.server.regions": "选择可以为调度域名提供服务的区域。",
-	"edgeHostname.update.accepted.modal.content": "边缘主机名更新已接受。更改需要几秒钟才能生效。您可以在 {history} 下拉列表中检查状态。",
+	"edgeHostname.update.accepted.modal.content": "调度域名更新请求已接受。调度域名的部署需要几分钟时间。您可以打开详请页面并在 {history} 下拉列表中检查部署状态。请确保域名已部署成功且配置生效，才在DNS记录中使用该调度域名。",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:785]: Please replace {history} with {操作历史}.

When the language selected by user is Chinese, there won't be the label for the dropdown won't History.

Comment (jungho73) [reply zh-Hans.json:785]: As you can see when you select in Chinese, the label will use this below.
"button.menu.text.deployment.history": "部署历史",
That means  {history} will be converted to  "部署历史".

But do you mean to want to change  "部署历史" to "操作历史", right?
if it is right, I will add new item for {history} edgehostname page.

Comment (Jianghai-Quantil) [reply zh-Hans.json:785]: @jungho73 Either "部署历史" or "操作历史" is fine. Let's use "部署历史" to be consistent with the label. 

Don't know that {history} is a variable to be expanded. Had thought {} is used for styling. 

Then no change is required. Approving the PR.

**Review by Jianghai-Quantil — APPROVED** [review]
Good to merge.

**Pull request merged**